core/include/traccc/clusterization/clusterization_algorithm.hpp

```diff
@@ -9,26 +9,18 @@
 
 #include "traccc/edm/cell.hpp"
 #include "traccc/edm/cluster.hpp"
-#include "traccc/edm/internal_spacepoint.hpp"
 #include "traccc/edm/measurement.hpp"
 #include "traccc/edm/spacepoint.hpp"
 #include "traccc/geometry/pixel_data.hpp"
 
 // clusterization
 #include "traccc/clusterization/component_connection.hpp"
 #include "traccc/clusterization/measurement_creation.hpp"
-#include "traccc/clusterization/spacepoint_formation.hpp"
-
-#ifdef _OPENMP
```

Comment:
If we removed the OpenMP include from here, then each example from the `openmp` folder should include it.

Reply:
Yeah I included it in each example file